src/routes/vipps.ts

```diff
@@ -247,6 +247,18 @@ router.post(
   }
 );
 
+router.get("/agreement/:urlcode/donations", async (req, res, next) => {
+  try {
+    const donations = await DAO.vipps.getDonationsByUrlCode('agr_4tswtyB') 
```

Comment:
The agreement code is hard coded, needs to be fetched from url params

https://expressjs.com/en/guide/routing.html -> Route parameters heading